### ISSUE TITLE
Font readability issue in documentation

### ISSUE THREAD

**Issue by vladshcherbin** [sveltejs/svelte.dev#552]
First of all, thank you for the incredible work on Svelte 5! I’ve been really excited about this release and checking for updates every few days 😅

That said, I’m really struggling with the new serif font in the docs. It’s so narrow and light that even getting through the first Overview page feels tough. The words tend to blend together which makes reading a challenge.

I get that font choices are often based on personal preferences, but my concern is really about readability. I’d like to suggest keeping the headers in serif and using sans-serif for the body text to make it easier to read.

I know browser extensions can change the font, but the thought of having to use one honestly makes me deeply sad.

Thank you again for all the amazing work and for considering this ♥️

<details>
 <summary>Current docs screenshot in dark mode</summary>

<img width="1470" alt="image" src="https://github.com/user-attachments/assets/e2ba0529-b58c-44c7-8ef0-74422980b6ff">

</details>

Related:

- https://github.com/sveltejs/svelte.dev/issues/174
- https://github.com/sveltejs/svelte.dev/issues/517 (30+ upvotes)

**Comment by dyc3** [comment]
I second this experience. Serif fonts are particularly hard to read for people with dyslexia. The [British Dyslexia Association recommends](https://www.bdadyslexia.org.uk/advice/employers/creating-a-dyslexia-friendly-workplace/dyslexia-friendly-style-guide) sans-serif fonts like Arial and Helvetica.

It would be really nice to have an accessibility option to swap out the font with a sans-serif font. This _can_ be worked around by users (custom stylesheet, browser extensions, etc etc) but it would be nice to have it built in.

**Comment by dummdidumm** [comment]
Could you specify if you're using dark mode or light mode, and whether or not you're using a high DPI screen? (We fall back to Georgia on lower resolution screens)

**Comment by dyc3** [comment]
I'm currently on a 16:9 1080p screen, in both dark and light mode.

**Comment by vladshcherbin** [comment]
I’m using a MacBook Air with a 2560 × 1664 liquid retina display (224 PPI) in dark mode. The first message includes a screenshot showing how it looks on my screen 🙌

**Comment by rmunn** [comment]
https://github.com/sveltejs/svelte.dev/issues/517#issuecomment-2432549124 is an excellent comment on this issue, and I'll quote the most important parts here (thanks @saminton for beating me to that comment by a half hour; I was going to say pretty much exactly that, but instead I'll just quote you):

> Sans-serif fonts were designed to be more easily read on screens. While monitors have come a long way, I—like many other developers—am not reading software documentation on a smartphone or a high-DPI monitor. I have a standard pixel density of 92 PPI.
> 
> At this pixel density, small Sans-serif fonts are legitimately more readable. Serif fonts have more detail and nuance, which becomes harder to discern due to the low pixel count.
> 
> ...
> 
> This is not a discussion about design or personal preference but one about accessibility. If you have any respect for accessibility, which you should have, then the body font should be switched to Sans-serif.

In @Rich-Harris's ["Svelte 5: New and Improved" presentation](https://www.youtube.com/watch?v=aFFYETUdTxk), around 20:45, Rich asks about the font choices, "Do people like this? Is it readable? Does it look nice?"

The answer seems to be no. For myself, I'd qualify that a little: it _does_ look nice, **if** you have a high-DPI screen or good eyes. But for most people, it's not very readable — and for some people with eyesight problems such as astigmatism, it's not readable **at all**. Case in point: https://github.com/sveltejs/svelte.dev/issues/517.

I am adding my voice to the "please switch back to sans-serif, because there's a good reason it's used everywhere on the Web" choir.

**Comment by notramo** [comment]
I like the idea of switching to serif font. The problem is not that @Rich-Harris likes the serif font. The problem is that it's a terrible serif font. It has weird kerning and is not optimized for print neither for screen. It's optimized for nothing. It belongs in a trash bin. There are many amazing serif fonts, with cool vibes. There is a great argument made here [What the font?](https://svelte.dev/blog/the-omnisite#What-the-font) why the serif font was chosen. However this font does not bring the vibe described there. It brings the vibe of **broken webpages**, when a web font could not load for some reason. At least, on my Linux system, it resembles the default fallback font so closely I had to check why the page font failed to load. Turns out, it loaded successfully. So, in order to give the analog, unconventional vibe, please choose a readable serif font.

It's worth noting, that Anthropic, a leader in one of the most innovative and modern fields chose a serif font for [Claude](https://claude.ai/). They simply provide a setting to set it to OpenDyslexic:
![image](https://github.com/user-attachments/assets/f0aaa56b-c1e4-406a-9346-3634d5776fa7)

[Inflection](https://inflection.ai/) also chose a serif font for [Pi](https://pi.ai/).

Both of these models are praised by many for their warm and personal tone. While the phrasing of the model output is mainly responsible for it, I think the serif font and colorscheme in the UI also adds to it.

So if Rich wants to follow similar wibes with the website, I support the idea.
But please:
- chose a normal font, one that has actually the vibes that the site is aiming for
- add option to switch it

**Comment by rmunn** [comment]
I'd like to vote against OpenDyslexic as a default. I've always thought it looked hideous. It's well designed for what it's intended to do — break up the symmetry of (most) letters so people with dyslexia have an easier time reading them — so kudos to its designers. The problem is that the very thing it needs to do in order to achieve its purpose, breaking up the symmetry of most letters, is also what makes it ugly. So for those who don't need it, it's even worse than the current serif font.

If we allow users to choose their fonts, then certainly OpenDyslexic is a good choice to have on there, for the people who will benefit from it. But it should not be the default, or it will put off many people who see the site for the first time.

**Comment by maidzen** [comment]
I came here to post the same issue, i have Dyslexia and i have problems reading the Docs.

I understand that im part of a minority with this Problem but on the other hand, for Docs, readability should always be the priority over Design. 

I fixed the problem on my personal Computers with css Browser Extensions but, on my work pcs im not allowed to use Extensions.

I don't even need a new Default, If i could change the font like changing the Dark/Light theme i would be more then happy.
